src/webnn_native/dml/GraphDML.cpp

```diff
@@ -754,9 +754,9 @@ namespace webnn_native { namespace dml {
         ::dml::Expression padding = mExpression.at(inputsOperand[1].Get());
 
         // Workaround(mingming): If padding was added in mGraph, it must be used.
-        // Use time(NULL) to generate a unique name for the output node.
+        // Use (time(NULL) + mExpression.size()) to generate a unique name for the output node.
         // This may be a dml issue: https://github.com/microsoft/DirectML/issues/133.
-        std::string name = std::to_string(time(NULL));
+        std::string name = std::to_string(time(NULL) + mExpression.size());
```

Comment:
web can't make sure `time(NULL)`  is unique, so how about generating the identify like openvino? 
`std::string name = “pad” + mExpression.size();`

Reply:
Good suggestion, thanks! @fujunwei